File: banda/JCZhang-Banda/JCZhang-Banda-JAVA/src/main/java/models/Percussionist.java
package models;

import interfaces.IPercussionist;

public class Percussionist extends Musician implements IPercussionist {
    String name;
    int yearsOfExperience;
    String percussionType;

    static double salary = Musician.salary*1.35;
    public Percussionist(String name, int yearsOfExperience, String percussionType) {
        super(name, yearsOfExperience);
        this.name = name;
        this.yearsOfExperience = yearsOfExperience;
        this.percussionType = percussionType;
    }


    @Override
    public void toPercussion() {
        System.out.println("Im a Percussionist and im playing Guitar");
    }
}

File: banda/JCZhang-Banda/JCZhang-Banda_KOTLIN/src/main/kotlin/models/Percussionist.kt
package models

import interfaces.IPercussionist

class Percussionist(override var name: String?, override var yearsOfExperience: Int, var percussionType: String) :
    Musician(name, yearsOfExperience), IPercussionist {
    override fun toPercussion() {
        println("Im a Percussionist and im playing Guitar")
    }

    companion object{
        var salary: Double = Musician.salary * 1.35
    }

}